src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.Custom.pkgdef

```diff
@@ -54,3 +54,9 @@
 "Value"=dword:00000001
 "Title"="Experimental Razor project multi-targeting support (requires restart)"
 "PreviewPaneChannels"="*"
+
+[$RootKey$\FeatureFlags\Razor\LSP\UsePreciseSemanticTokenRanges]
+"Description"="Use precise semantic token ranges when requesting semantic token information on razor files. Note: This is experimental, some actions may not apply correctly, or at all!"
+"Value"=dword:00000001
+"Title"="Use precise semantic token ranges when requesting semantic token information on Razor files (requires restart)"
+"PreviewPaneChannels"="IntPreview,int.main"
```

Comment:
@davidwengier I specified both intpreview and intmain here based which is what `ShowAllCSharpCodeActions` is also doing. I'm assuming `*` means pick all?

Reply:
Yeah, thats correct. If you want the feature flag to only be togglable by Microsoft employees, then what you have is correct. If you want users to be able to toggle it then it would need to change. For `IncludeProjectKeyInGeneratedFilePath` I wanted it user togglable so people had a get-out-of-jail action they could take if things go wrong, rather than wait for servicing. In theory we can also use a mechanism to change the feature flag value remotely, but I don't know how to do that :)